meson.build

```diff
@@ -450,6 +450,10 @@ if get_option('buildtype') != 'debug'
         possible_link_flags += '-Wl,--gc-sections'
 endif
 
+if get_option('mode') == 'developer'
+        possible_cc_flags += ['-fno-omit-frame-pointer']
+endif
```

Comment:
Not sure this is conditionalized. If it is useful to get stacktrace, then should this be enabled unconditionally?
E.g. fedora provides debuginfo packages but built with `-mode=release`.
cc @keszybz ?

Reply:
Whether to do this unconditionally should be a distro decision. I'm actually drafting a change proposal for Fedora to build with -fno-omit-frame-pointer by default, but the decision should still be left to the distro I believe.

Reply:
I have to admit that I don't know how this works exactly. We get the `-debuginfo` packages and they seem to work, we get a full backtrace with all details. But if something is worth changing there, I agree that a Change proposal is the way to go.